acceptance/helpers/machine/machine.go

```diff
@@ -63,6 +63,17 @@ func (m *Machine) TargetOrg(org string) {
 	ExpectWithOffset(1, out).To(MatchRegexp("Currently targeted namespace: " + org))
 }
 
+func (m *Machine) DeleteNamespace(namespace string) {
+	By("deleting a namespace")
+
+	out, err := m.Epinio("", "namespace", "delete", "-f", namespace)
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), out)
+
+	namespaces, err := m.Epinio("", "namespace", "list")
+	ExpectWithOffset(1, err).ToNot(HaveOccurred())
+	ExpectWithOffset(1, namespaces).ToNot(MatchRegexp(namespace))
```

Comment:
Alternative, should we wish to avoid `list` ... `show`, and expect error.

Reply:
07024aa1f3842da96c806101142960a3b20e4c24